app/jobs/full_text_search/update_issue_content_job.rb

```diff
@@ -47,7 +52,14 @@ def create_content(issue_id, excludes: [])
                 .reject {|j| j.notes.blank? || excludes.include?(j.id) }
                 .sort_by(&:id)
                 .map(&:notes)
+      attachment_contents = issue.attachments
+                              .order(:id)
+                              .reject {|a| excludes.include?(a.id) }
+                              .collect {|a| [a.filename, a.description] }
+                              .flatten
+                              .compact
       content.concat(notes)
+      content.concat(attachment_contents)
```

Comment:
How about adding to `content` directly instead of `.flatten.compact`?

```ruby
issue.attachments.order(:id).each do |attachment|
  content << attachment.filename if attachment.filename.present?
  content << attachment.description if attachment.description.present?
end
```